modules/Core/pages/validate.php

```diff
@@ -29,10 +29,11 @@
             $user,
         ));
 
-        GroupSyncManager::getInstance()->broadcastChange(
+        GroupSyncManager::getInstance()->broadcastGroupChange(
             $user,
             NamelessMCGroupSyncInjector::class,
-            [$user->getMainGroup()->id]
+            [$user->getMainGroup()->id],
+            [],
```

Comment:
Same as above but this one can be removed, broadcastGroupChange is already triggered in the hook above